data/json/items/comestibles/brewing.json

```diff
@@ -316,7 +316,7 @@
         "charges" : 1,
         "fun" : -12,
         "brewable" : {
-            "time" : 7200,
+            "time" : 2400,
```

Comment:
Looks good. Have you considered converting `time` to hours to be consistent with `spoils`?